TabCode.cabal

```diff
@@ -45,6 +45,20 @@ executable tc2mei
   hs-source-dirs: prog
   ghc-options: -O2 -threaded
 
+executable tc2mei-js
+  build-depends: base >= 4.8 && < 5
+               , bytestring >= 0.10.8.1 && < 0.11
+               , optparse-applicative >= 0.14 && < 0.15
```

Comment:
This executable target doesn't use optparse-applicative. You should be able to remove this.